modules/connected-to-disconnected-restore.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-restore_{context}"]
+= Restoring the network
+
+In the event that you want to make your disconnected cluster connected again, in order to start pulling images from your online registries, you only need to delete the `ImageContentSourcePolicy` (ICSP) objects you created when you disconnected the cluster. Without the ICSP, pull requests to external registries are no longer redirected to the mirror registry.
```

Comment:
Consider: "If you want to reconnect a disconnected cluster and pull images from online registries, delete the cluster's `ImageContentSourcePolicy` (ICSP) objects."